Gaijinizer patch/Map070.txt

```diff
@@ -1680,20 +1743,19 @@ I was threatened. I am human.#MTLed
 
  古風な妖怪って感じだろ？
 > CONTEXT: Map070/events/9/pages/1/38/Dialogue/enemy7
-At the corner of a road without lights in the
-middle of the night. I liked to watch my opponent
-bumble and run away to me. You feel like an
-old-fashioned monster?#MTLed
+On the corner of a road without street lights in
+the middle of the night, I liked seeing my
+opponent run away all scared. It kinda felt like
+being an old-fashioned monster, you know?#RonIffy
```

Comment:
Here's a rewording that sounds a bit better:
>_**Standing on a dark corner of the street
in the middle of the night... It was fun watching
them run away in terror. It kinda felt like being
one of those old-fashioned monsters, you know?**_